Tests/store_packs.py

```diff
@@ -0,0 +1,512 @@
+import json
+import os
+import sys
+import argparse
+import warnings
+import shutil
+import uuid
+import google.auth
+from google.cloud import storage
+from distutils.util import strtobool
+from distutils.version import LooseVersion
+from datetime import datetime
+from zipfile import ZipFile, ZIP_DEFLATED
+from Tests.test_utils import run_command, print_error, print_warning, print_color, LOG_COLORS, \
+    collect_content_items_data, input_to_list
+
+# global constants
+STORAGE_BASE_PATH = "content/packs"
+IGNORED_PATHS = ['Packs/__init__.py']
+# the format is defined in issue #19786, may change in the future
+DIR_NAME_TO_CONTENT_TYPE = {
+    "Classifiers": "classifier",
+    "Dashboards": "dashboard",
+    "IncidentFields": "incidentfield",
+    "IncidentTypes": "incidenttype",
+    # "IndicatorFields": "indicatorfields",
+    "Integrations": "integration",
+    "Layouts": "layout",
+    "Playbooks": "playbook",
+    "Reports": "report",
+    "Scripts": "automation",
+    "Widgets": "widget"
+}
+
+
+class Pack(object):
+    """Class that manipulates and manages the upload of pack's artifact and metadata to cloud storage.
+
+    Args:
+        pack_name (str): Pack root folder name.
+        pack_path (str): Full path to pack folder.
+
+    Attributes:
+        PACK_INITIAL_VERSION (str): pack initial version that will be used as default.
+        DATE_FORMAT (str): date format of.
+        CHANGELOG_JSON (str): changelog json full name, may be changed in the future.
+        CHANGELOG_MD (str): changelog md full name.
+        README (str): pack's readme file name.
+        METADATA (str): pack's metadata file name, may be changed in the future.
+        INDEX_NAME (str): pack's index name, may be changed in the future.
+
+    """
+    PACK_INITIAL_VERSION = "1.0.0"
+    DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+    CHANGELOG_JSON = "changelog.json"
+    CHANGELOG_MD = "changelog.md"
+    README = "README.md"
+    METADATA = "metadata.json"
```

Comment:
is this the me metadata in the content repo or the uploaded one?

Reply:
Currently they have same name, we need to change the name of one in repo